### DISCUSSION TITLE
[@types/node] test-runner run->setup signature was changed in 21.6.0

### DISCUSSION THREAD

**Discussion by koshic** [DefinitelyTyped/DefinitelyTyped#68209] [Issues with a @types package]
After https://github.com/nodejs/node/pull/50982 fix landed, `setup?: (root: Test) => void | Promise<void>;` should be `setup?: (testStream: TestsStream) => void | Promise<void>;`, as it according to the docs.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
This fix wasn't landed to v20.x

**Reply by koshic** [reply]
So you're saying that we should wait for 22.x, right?

**Reply by Semigradsky** [reply]
Yeah, currently we haven't `@types/node` for non-LTS versions. You can also push nodejs team to apply fix to v20.x

**Reply by koshic** [reply]
https://github.com/nodejs/node/pull/50982#issuecomment-1893703885
